CHANGELOG.rst

```diff
@@ -5,6 +5,13 @@ Changelog
 This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
 
+1.0.0-alpha.6-SNAPSHOT (2021-04-x)
+
+**Fixed**
+
+* Prices are now sorted more intuitively
+
+
```

Comment:
Why are you introducing the new snapshot here? Why not put it into the .5 snapshot?
Also if we release the alpha.5 version such changes should be done in a branch preparing development for the current version